champion.py

```diff
@@ -30,43 +43,56 @@ class BaseChampion:
         ...
     """
 
-    def __init__(self, champion_name: str):
-        self.initialize_base_stat(champion_name)
+    def __init__(self, champion_name: str, level: int):
+        assert level in np.arange(1, 19), "Champion level should be in the [1,18] range"
```

Comment:
why no
```suggestion
        assert 1 <= level <= 18
```

Reply:
I wanted to limit the level to an **integer** between 1 and 18. what else do you suggest ? isnstance(level,int) and  1<= level <= 18 ?

Reply:
doesn't the annotation `level: int` take care of that?

Reply:
In python we use a type **hinting** system ... ^^'